s3/pkg/service/multipart.go

```diff
@@ -257,7 +261,24 @@ func (s *s3Service) UploadPart(ctx context.Context, stream pb.S3_UploadPartStrea
 		log.Errorln("failed to upload part to backend. err:", err)
 		return err
 	}
-	uploadResponse.ETag = result.ETag
```

Comment:
IMO, it's better to get ETag from the response, and check if it null, and it it is not null, check if it is the same as the one calculated here.

Reply:
@sfzeng , Thanks for your viewing,  I have passed ETag from client to backend, and the check of ETag should be done by backend.  because different backend calculate ETag with different ways,  the validity of data should be done at backend

Reply:
But it sees when sd.UploadPart is called, ETag has not calculated yet, so the backend driver cannot get the ETag?